docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
+
+.. task:: enable_flocker_control
+   :prompt: [root@control-node]#
+
+The control service needs to accessible remotely.
+To configure firewalld to allow access to the control service REST API, and for agent connections,
 
 .. task:: open_control_firewall
 
+(On AWS, an external firewall is used instead, which will need to be configured similarity).
```

Comment:
This is just AWS, the other providers don't have an external firewall. I'm not going to file a issues for this: it isn't clear to me that this belongs in our documentation (we can't document all the details about how to use every cloud provider). If it is in scope, it can be filed as part of a specific user story.